pkg/cmd/server/start.go

```diff
@@ -210,10 +213,15 @@ func start(cfg *config, args []string) error {
 
 		assetAddr := net.JoinHostPort(cfg.MasterAddr.Host, strconv.Itoa(cfg.BindAddr.Port+1))
 
+		// always include the all-in-one server's web console as an allowed CORS origin
+		// always include localhost as an allowed CORS origin
+		cfg.CORSAllowedOrigins = append(cfg.CORSAllowedOrigins, assetAddr, "localhost")
```

Comment:
No it means localhost is added to the list of Origins that will result in CORS headers being returned, necessary for the web console to talk to the API server.  That line adds both localhost and assetAddr where assetAddr is currently defined as whatever you configured for the master's hostname:port + 1 on the port.   This list of allowed origins can always be added to by specifying the --cors-allowed-origins flag during openshift start